src/coreclr/System.Private.CoreLib/src/System/RuntimeHandles.cs

```diff
@@ -74,7 +118,7 @@ internal bool IsNullHandle()
 
         internal TypeHandle GetNativeTypeHandle()
```

Comment:
Should we delete this instead and change the callers to call `GetRuntimeType().GetNativeTypeHandle()` to ensure that there is null check if necessary?

Reply:
Good idea.